src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogSenderFlowNode.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.plugins.opentelemetry.job.log;
+
+import jenkins.util.JenkinsJVM;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import java.io.IOException;
+
+/**
+ * Sender for logs retaled to a flow node.
+ * See https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin/blob/master/src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchSender.java#L108
+ */
+final class OtelLogSenderFlowNode extends OtelLogSender {
```

Comment:
Do we need today to differentiate OtelLogSenderFlowNode from OtelLogSenderRoot?

Reply:
no, I have to delete the two classes after merge your changes